lib/Doctrine/Migrations/Tools/Console/Command/DoctrineCommand.php

```diff
@@ -75,6 +92,26 @@ protected function initialize(InputInterface $input, OutputInterface $output) :
             $this->dependencyFactory->setConfigurationLoader($configurationLoader);
         }
 
+        if ($input->getOption('em')!== null && $input->getOption('conn')!==null) {
+            throw new InvalidOptionUsage('You can specify only one of the --em and --conn options.');
+        }
+
+        $connectionLoader = $this->dependencyFactory->getConnectionLoader();
+        if ($connectionLoader instanceof ConnectionRegistryConnection && $input->getOption('conn')!==null) {
+            $connectionName = $input->getOption('conn');
+            $connectionLoader->setConnectionName((string) $connectionName);
```

Comment:
At this point `$input->getOption('conn')` can't return a null value.

Reply:
same as https://github.com/doctrine/migrations/pull/1023#discussion_r446008824